src/hotspot/share/runtime/arguments.cpp

```diff
@@ -1437,7 +1437,6 @@ void Arguments::set_mode_flags(Mode mode) {
     break;
   case _comp:
     UseInterpreter           = false;
-    BackgroundCompilation    = false;
```

Comment:
Disabling `BackgroundCompilation` when -Xcomp is specified is not the right
answer here. When `-Xcomp` is specified, `-Xbatch` needs to be turned off
and I believe that's what old L1440 is doing.

Reply:
Dan is right, you  can't change the definition of -Xcomp.

But `-Xbatch` is `-XX:-BackgroundCompilation`. So the second Dan's statement is incorrect.